settings/testfiles/test.env

```diff
@@ -11,3 +11,16 @@ AIRFLOW_VAR_MYVAR=myval
 AIRFLOW_CONN_LOCAL_POSTGRES=postgres://username:password@example.db.example.com:5432/schema
 AIRFLOW_VAR_MYVAR=myval
 AIRFLOW_CONN_LOCAL_POSTGRES=postgres://username:password@example.db.example.com:5432/schema
+
+AIRFLOW_VAR_MYVAR=myval
+AIRFLOW_CONN_LOCAL_POSTGRES=postgres://username:password@example.db.example.com:5432/schema
+AIRFLOW_VAR_MYVAR=myval
```

Comment:
Do you intend to commit this change. In one of your other PRs, this was deleted.